spotify/index.js

```diff
@@ -2834,3 +2846,7 @@ ControllerSpotify.prototype.startVolumeTimerLimit = function () {
         apiCallsCounter = 0;
     }, apiCallsTimespan);
 };
+
+ControllerSpotify.prototype.handleBrowsingError = function (errorMsg) {
```

Comment:
This should not be necessary if you reject the promise, there will be a toast

Reply:
ok in the current version you forgot to remove this from `catch` from all `getXXX` methods, so it crashed due it couldn't find this method. Need to remove all references then.

Reply:
https://github.com/volumio/volumio-plugins-sources/pull/262